test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -53,21 +57,22 @@ public void CreatePackageWithFiles()
                 {
                     var xml = XDocument.Load(stream);
                     var itemGroup = @"<ItemGroup>
-                    <None Include=""abc.props"" Pack=""True""  PackagePath=""build"" />
+                    <None Update=""abc.props"" Pack=""True""  PackagePath=""build"" />
                     <None Include=""abc.dll"" Pack=""True""  PackagePath=""lib"" />
                     <Content Include=""abc.txt"" Pack=""True"" />
-</ItemGroup>";
-                    ProjectFileUtils.SetTargetFrameworkForProject(xml, "TargetFramework", Constants.ProjectTargetFramework);
+                    </ItemGroup>";
+                    ProjectFileUtils.SetTargetFrameworkForProject(xml, tfmProperty, tfmValue);
                     ProjectFileUtils.AddCustomXmlToProjectRoot(xml, itemGroup);
                     ProjectFileUtils.WriteXmlToFile(xml, stream);
                 }
 
                 _dotnetFixture.RestoreProjectExpectSuccess(workingDirectory, projectName, testOutputHelper: _testOutputHelper);
-                _dotnetFixture.PackProjectExpectSuccess(workingDirectory, projectName, $"/p:PackageOutputPath={workingDirectory}", testOutputHelper: _testOutputHelper);
+                var results = _dotnetFixture.PackProjectExpectSuccess(workingDirectory, projectName, $"/p:PackageOutputPath={workingDirectory}", testOutputHelper: _testOutputHelper);
 
                 var nupkgPath = Path.Combine(workingDirectory, $"{projectName}.1.0.0.nupkg");
                 var nuspecPath = Path.Combine(workingDirectory, "obj", $"{projectName}.1.0.0.nuspec");
 
+                Assert.DoesNotContain("NU5118", results.AllOutput);
```

Comment:
Should this now say Assert.DoesNotContain("NU5", results.AllOutput)

That way it's a bit more future proof against warnings.